utils_test.go

```diff
@@ -0,0 +1,60 @@
+package orientation
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+)
+
+func TestF3(t *testing.T) {
+	F3Img, err := getImageFromFilePath("./TestImages/F-3.jpg")
+	if err != nil {
+		fmt.Printf("error: %s", err)
+	}
+	imgToSaveBytes, orientation := ReadImage(F3Img, logrus.NewEntry(logrus.StandardLogger()))
+
+	os.WriteFile("./TestImages/f3_after_tests.jpg", imgToSaveBytes, os.ModeDevice.Perm())
+	if orientation == "none" {
```

Comment:
How do you find removed that if and use https://pkg.go.dev/github.com/stretchr/testify/assert#NotEqual
`NotEqual(t TestingT, "none", orientation, "test F3 passed with orientation %s")`